semaphoreci.sh

```diff
@@ -119,10 +119,10 @@ alltests() {
 testsuite() {
     ## Run just the compiler testsuite.
     cd ${SEMAPHORE_PROJECT_DIR}/build
-    make -j$(nproc) check-gcc-d
 
     # Other tests that check parts of the compiler.
     make check-gcc RUNTESTFLAGS="help.exp"
+    make -j$(nproc) check-gcc-d
```

Comment:
We're going to have to clean up the comments here too. :-)

Reply:
Can we just remove "# Other tests that check parts of the compiler."? the first line already says that this is the compiler testsuite.

Reply:
Yes, go for it.